### DISCUSSION TITLE
Discord community call: 20 January 2024

### DISCUSSION THREAD

**Discussion by BenJamesBen** [freesewing/freesewing#5717] [Support] [answered]
You are all invited to our Discord hangout.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This page will be updated after the call with notes.

### About

FreeSewing's Discord hangout is an informal hangout/chat/meeting that is open to everyone.
The call is organized/hosted by @tangerineshark.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

### Details

Location: https://discord.freesewing.org/ (meeting room: 🔊voice chat)
Date: January 20th, 2024 5pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/jan-20-2024/5pm).
Or check the Events tab in Discord!
The meetings will always be at 6pm, Antwerp time.

### Agenda

- Discuss any updates on pattern designs https://github.com/freesewing/freesewing/discussions?discussions_q=is%3Aopen+label%3A%22%3Ashirt%3A+pattern+design%22
  - This could be any patterns that someone has made or comments on features to add to specific patterns
- See if anyone wants to try looming their own material
  - Either at a local maker space
  - or 3d print their own https://github.com/GliaX/gauzeloom
- Ideas for what to do with scrap material from making clothing

**Comment by darigovresearch** [comment]
Is it possible to add the following to the next call?

- Discuss any updates on pattern designs https://github.com/freesewing/freesewing/discussions?discussions_q=is%3Aopen+label%3A%22%3Ashirt%3A+pattern+design%22
  - This could be any patterns that someone has made or comments on features to add to specific patterns
- See if anyone wants to try looming their own material 
  - Either at a local maker space
  - or 3d print their own https://github.com/GliaX/gauzeloom
- Ideas for what to do with scrap material from making clothing

**Comment by BenJamesBen** [comment, marked as answer]
January 20, 2024, meeting notes

- What people have been working on: Embroidery, Making shoe lasts from a 3d scan, repairing jeans and designing an eye patch.
- A way to re-sew an outer seam on a jacket that came undone? Hand stitch with a ladder/slip stitch.
- Pattern designs: Updates were made to the eye patch, tailor's ham, and chicken pattern weight discussions. These GitHub discussions are to help discussion and take input, to help design iteration, and an excuse to test making patterns. Will add to April 27 agenda to review status and close inactive discussions.
- How to decide how to allocate time between coding or sewing?
- Looming your own material or 3d print them? 
- Ideas for using fabric scraps? Stuffing, color blocking, inside pockets, clothing repair, recycling, composting?, 
- Adding pockets to existing skirts, tablet looms.

**Comment by darigovresearch** [comment]
Thanks for the notes @BenJamesBen, will mark your notes as the answer unless anyone has any objections
